Documentation/TopLevelObjects/Other.rst

```diff
@@ -1,91 +1,135 @@
-.. index::
-   Top-level objects; temp
-   Top-level objects; styles
-   Top-level objects; lib
-.. _top-level-objects-other-reserved-tlo-s:
-.. _top-level-objects-temp:
-.. _tlo-temp:
-.. _top-level-objects-styles:
-.. _tlo-styles:
-.. _top-level-objects-lib:
-.. _tlo-lib:
+..  include:: /Includes.rst.txt
+..  _top-level-objects-other-reserved-tlo-s:
 
-==================
-temp, styles & lib
-==================
+==========================
+Reserved top level objects
+==========================
 
-.. container:: table-row
 
-   Property
-         temp
+..  index:: Top-level objects; temp
 
-   Data type
-         *(whatever)*
+..  _top-level-objects-temp:
+..  _tlo-temp:
 
-   Description
-         This top-level object name is reserved.
+temp
+====
 
-         :typoscript:`temp` and :typoscript:`styles` are used for code-libraries you can
-         copy during parse-time, but they are not saved with the template in
-         cache. :typoscript:`temp` and :typoscript:`styles` are unset before the template is
-         cached! Therefore use these names to store temporary data.
+..  confval:: temp
+    :name: tlo-temp
 
-         :typoscript:`lib` can be used for a "library" of code, you can reference in
-         TypoScript (unlike :typoscript:`styles` which is unset).
+    This top-level object name is reserved.
 
+    The top level object :typoscript:`temp` is used to store and copy
+    TypoScript code during parse time.
 
-.. index:: Top-level objects; tt_content
-.. _tlo-tt:
-.. _tlo-tt_content:
+    :typoscript:`temp` is unset before the template is cached, objects in it
+    can therefore not be referenced. Use :confval:`tlo-lib` for that purpose.
 
+Example: Use the top level object `temp` to copy code
+-----------------------------------------------------
 
-==========
-tt_content
-==========
+..  code-block:: typoscript
+    :caption: EXT:my_sitepackage/Configuration/TypoScript/setup.typoscript
 
-The top level keyword :typoscript:`tt_content` is used to render content from the table :sql:`tt_content`.
+    temp.some_content = TEXT
+    temp.some_content.value = Hello World!
 
+    // Output
+    // <h1>Hello World!</h1><p>Hello World!</p>
+    page = PAGE
+    page {
+        10 < temp.some_content
+        10.wrap = <h1>|</h1>
 
-.. index:: Top-level objects; resources
-.. _top-level-objects-resources:
-.. _tlo-resources:
+        20 < temp.some_content
+        20.wrap = <p>|</p>
+    }
 
-=========
-resources
-=========
+..  index:: Top-level objects; lib
 
-.. container:: table-row
+..  _top-level-objects-lib:
+..  _tlo-lib:
 
-   Property
-         resources
+lib
+===
 
-   Data type
-         readonly
+..  confval:: lib
+    :name: tlo-lib
 
-   Description
-         Resources in list (internal)
+    This top-level object name is reserved.
 
+    The top-level object :typoscript:`lib` is used to store, copy and reference
+    TypoScript code.
 
+    Other than :confval:`tlo-temp` is not unset before the template is cached,
```

Comment:
This sentence is hard to understand (at least for me), I read it multiple times.

Reply:
How would you formulate it?